doc/io.rst

```diff
@@ -635,6 +635,31 @@ For example:
     Not all native zarr compression and filtering options have been tested with
     xarray.
 
+Consolidated Metadata
+~~~~~~~~~~~~~~~~~~~~~
+
+Xarray needs to read all of the zarr metadata when it opens a dataset.
+In some storage mediums, such as with cloud object storage (e.g. amazon S3),
+this can introduce significant overhead, because two separate HTTP calls to the
+object store must be made for each variable in the dataset.
+With version 2.3, zarr will support a feature called *consolidated metadata*,
+which allows all metadata for the entire dataset to be stored with a single
+key (by default called ``.zmetadata``). This can drastically speed up
+opening the store. (For more information on this feature, consult the
+`zarr docs <https://zarr.readthedocs.io/en/latest/tutorial.html#consolidating-metadata>`_.)
+
+If you have zarr version 2.3 or greater, xarray can write and read stores
+with consolidated metadata. To write consolidated metadata, pass the
+``consolidate=True`` option to the
+:py:attr:`Dataset.to_zarr <xarray.Dataset.to_zarr>` method::
+
+    ds.to_zarr('foo.zarr', consolidate=True)
+
+To read a consolidated store, pass the ``consolidated=True`` option to
+:py:func:`~xarray.open_zarr`::
+
+    ds = xr.open_zarr('foo.zarr', consolidated=True)
```

Comment:
I wonder if we should try to use the same keyword argument name here -- it seems a little confusing to write `consolidated=True` when reading but `consolidate=True` when writing. Maybe both should be `consolidated=True`?

Reply:
I see your point, and I thought about this. I decided on what I have here because `consolidate` sounds more active and lets you know that some action, consolidation, will be performed on the store.

But I don't feel strongly about this and would be happy to change it.

Reply:
Anyone else have thoughts on this? @jhamman?

Reply:
My vote would be for using the same keyword in both contexts. So +1 for `consolidated`.